### DISCUSSION TITLE
新手的问题：nfo、fanart、poster能不能设置为只下载一个？

### DISCUSSION THREAD

**Discussion by aip5** [Yuukiy/JavSP#470] [问答 | Q&A]
### 问题详情

nfo、fanart、poster能不能设置为只下载一个？还有就是，刮削完后，视频文件名能不能直接自动设置为番号+片名，比如NSFS-060 女をその気にさせる 力づくの和姦3 妃ひかり.mp4；NSFS-060 女をその気にさせる 力づくの和姦3 妃ひかり-fanart.jpg

### 运行方式

我运行的是打包后的exe程序

### 代理

是

### 提交须知

- [X] 我确认使用的是最新版本并且阅读过Wiki帮助文档
- [X] 我确认已经搜索过Issue区，没有与我遇到的情况相同的Issue。

**Comment by glyh** [comment]
nfo和fanart和poster都是一定会生成的，extrafanart默认是被禁止的。poster其实就是fanart切割了一下。

至于文件名怎么起，参见[config.yml](https://github.com/Yuukiy/JavSP/blob/master/config.yml)中的各个basename_pattern
